app/assets/stylesheets/account/users_form.scss

```diff
@@ -0,0 +1,13 @@
+.rf-height-fullscreen {
```

Comment:
first 2 classes have nothing with forms
layout.scss

also rename user_form to form because it will contain general styles